persistence/src/main/scala/hmda/persistence/processing/HmdaFileParser.scala

```diff
@@ -115,6 +123,9 @@ class HmdaFileParser(submissionId: SubmissionId) extends HmdaPersistentActor wit
       log.debug(s"Parsing completed for $submissionId")
       publishEvent(ParsingCompleted(submissionId))
 
+    case CompleteParsingWithErrors =>
+      log.debug(s"Parsing completed for $submissionId, errors found")
```

Comment:
Are we missing a `publishEvent(ParsedWithErrors)` here so that it gets picked up by the event processor?